youtrack_sdk/client.py

```diff
@@ -403,3 +409,23 @@ def link_issues(
                 data=Issue(id=target_issue_id),
             ),
         )
+
+    def delete_issue_link(
+        self,
+        *,
+        source_issue_id: str,
+        target_issue_id: str,
+        link_type_id: str,
+    ):
+        """Delete the link between issues.
+
+        https://www.jetbrains.com/help/youtrack/devportal/operations-api-issues-issueID-links-linkID-issues.html#delete-Issue-method.
+
+        :param source_issue_id: Current issue ID
+        :param target_issue_id: The database ID of the issue that you need to unlink from the current one
+        :param link_type_id: The ID of the Issue link type
```

Comment:
The explanation still not clear to me. Maybe it's better to use "internal issue ID" and "internal issue link type ID"?

Reply:
They are referred to as database ID in the documentation
Those are exact words copied from the docs

https://www.jetbrains.com/help/youtrack/devportal/operations-api-issues-issueID-links-linkID-issues.html#delete-Issue-method

Reply:
Ok, but it doesn't help to understand what IDs should I use anyway :) Maybe instead of adding docstrings you can add a sentence to README about using internal IDs? Something like: "Prefer to use internal entity IDs everywhere. Some of the methods accept readable issue IDs (e.g. `HD-99`) but it's not supported everywhere." or similar.

Reply:
I agree